packages/trees/frontend/sirius-components-trees/src/views/TreeView.types.ts

```diff
@@ -16,42 +16,24 @@ import { TreeItemActionProps } from '../treeitems/TreeItemAction.types';
 
 export interface TreeViewComponentProps extends WorkbenchViewComponentProps {
   treeId: string;
+  tree: GQLTree;
```

Comment:
The TreeView component cannot be used directly as a workbench view anymore, it does not have to keep extending WorkbenchViewComponentProps it doesn't make sense. ExplorerView needs to do it (which it does) but not TreeView.